silkworm/db/test_util/temp_snapshots.hpp

```diff
@@ -376,4 +376,129 @@ class SampleTransactionSnapshotFile : public TemporarySnapshotFile {
               "03") {}
 };
 
+//! Empty Accounts KV segment file
+class EmptyAccountsDomainSegmentFile : public TemporarySnapshotFile {
```

Comment:
```suggestion
class SampleEmptyAccountsDomainSegmentFile : public TemporarySnapshotFile {
```
nit suggestion, because all other start with "Sample" prefix